src/components/StandardFacet.tsx

```diff
@@ -0,0 +1,116 @@
+import { FacetsProvider } from './Filters';
+import { FilterGroup } from './FilterGroup';
+import { Fragment } from 'react';
+import { DisplayableFacet } from '@yext/search-headless-react';
+import { FilterDivider } from './FilterDivider';
+import { BaseStandardFacetProps, BaseFacetCssClasses, BaseFacetProps } from './FacetProps';
+import { DisplayableFacetOption } from '@yext/search-core';
+
+/**
+ * The CSS class interface for {@link StandardFacet}.
+ *
+ * @public
+ */
+export interface StandardFacetCssClasses extends BaseFacetCssClasses {
+  standardFacetContainer?: string
+}
+
+/**
+ * Props for the {@link StandardFacet} component.
+ *
+ * @public
+ */
+export interface StandardFacetProps extends BaseFacetProps, BaseStandardFacetProps {
+  /** CSS classes for customizing the component styling. */
+  customCssClasses?: StandardFacetCssClasses
+}
+
+/**
+ * A component that displays a single standard facet.
+ *
+ * @param props - {@link StandardFacetProps}
+ * @param renderDivider - boolean
+ * whether to render a divider after the facet or not, defaults to false
+ * @returns A React component for the facet
+ *
+ * @public
+ */
+export function StandardFacet({ props, renderDivider = false }) {
+  const {
+    fieldId,
+    searchOnChange,
+    customCssClasses,
+  } = props;
+  return (
+    <FacetsProvider searchOnChange={searchOnChange} className={customCssClasses?.standardFacetContainer || ''}>
```

Comment:
Sorry this wasn't clear from the item description, but the intent was for this provider to live in the Facets component, not each of the singular Facet components